string system_state  # smart_cargo 현재 동작 상태
string door_state    # 적재함 Door 현재 동작 상태
string handler_state # Handler 현재 동작 상태
string docking_state # IF_docking Message 수신 상태
string type          # Message Object